include/yaml-cpp/node/convert.h

```diff
@@ -241,6 +242,35 @@ struct convert<std::list<T> > {
   }
 };
 
+// std::array
+template <typename T, std::size_t S>
+struct convert<std::array<T, S> > {
```

Comment:
There seems to be a few > > left in those files:
- include/yaml-cpp/exceptions.h
- include/yaml-cpp/node/convert.h
- include/yaml-cpp/node/detail/node_iterator.h
- test/node/node_test.cpp

I could fix them too in an other PR?